src/main.rs

```diff
@@ -139,6 +139,35 @@ enum AutofixMode {
     Yolo,
 }
 
+fn prelude(suggestion: &Replacement) {
+    let snippet = &suggestion.snippet;
+    if snippet.text.1.is_empty() {
+        if suggestion.replacement.ends_with('\n') || suggestion.replacement.starts_with('\n') {
+            println!("{}", "Insert line:".yellow().bold());
+        } else {
+            println!("{}", "At:".yellow().bold());
+            println!(
+                "{lead}{text}{tail}",
+                lead = indent(4, &snippet.text.0),
+                text = "⍉".red(),
```

Comment:
What's up with the [apl functional symbol circle backslash](http://graphemica.com/⍉) here? Do you mean to convey that the text is empty here? Then: Did you mean to use the [empty set](http://graphemica.com/∅)—or actually [ε](http://graphemica.com/ε)?

Reply:
I just wanted a pretty symbol. I would have preferred something like the google maps "you are here" marker. You know. with the pointy bottom and the round top.

Reply:
like, ▲ or ▼?

Reply:
idc. If you have sth you like, I'll use it.

Reply:
It's probably best to use something people actually have in their terminal font… so, v or ^? I don't mind either way

Reply:
if you want to change it, go ahead, otherwise i'm fine with merging with backslashed circle.

Reply:
I'll change it when I get back to a PC. Github web editing on mobile is a pain and would produce many commits ;)